susemanager-utils/susemanager-sls/src/modules/mgr_caasp_manager.py

```diff
@@ -229,7 +229,7 @@ def add_node(skuba_cluster_path,
     return ret
 
 
-def _upgrade_cluster_plan(skuba_cluster_path,
+def upgrade_cluster_plan(skuba_cluster_path,
```

Comment:
I didn't expose this method on purpose.

If you want to get the cluster upgrade plan, you should call `upgrade_cluster(plan=True, *args, **kwargs)`

Is there a reason why we do want to do it this way?

Reply:
I reverted it. It was just matter of personal preference to make the upgrade plan functionality more explicit.